.env

```diff
@@ -1,2 +1,3 @@
 APIHOST="http://localhost:1234/api"
 ENV="LOCAL"
+ENTITIES_LOGO_CDN="https://assets.cdn.io.italia.it/logos/organizations/"
```

Comment:
for consistency, please remove the last "/".
Add a small description for this variable in the readme file